corehq/apps/users/views/mobile/users.py

```diff
@@ -1379,7 +1379,9 @@ def _count_users(request, domain, user_type):
     (is_cross_domain, domains_list) = get_domains_from_user_filters(domain, user_filters)
     for current_domain in domains_list:
         if user_type == MOBILE_USER_TYPE:
-            user_count += count_mobile_users_by_filters(current_domain, user_filters)
+            user_count = count_mobile_users_by_filters(current_domain, user_filters)
+            group_count = len(load_memoizer(current_domain).groups)
+            user_count = (user_count, group_count)
```

Comment:
How would you feel about not having `user_count` variable store values for user_count and group_count?

Do you think it would be a good idea to rename the function and return a JSON response like -
```
return JsonResponse({
        'user_count': user_count,
        'group_count': group_count,
    })
```
And refactor the places where the API is consumed.

Reply:
This makes sense except for the fact that groups aren't applicable to web_users. So I would need to find a work around for that.